.github/workflows/manifests.yml

```diff
@@ -5,7 +5,8 @@ on:
   push:
   pull_request:
     paths:
-      - manifests/**/*.yml
+      - 'manifests/**/*.yml'
+      - '!manifests/3.**/**'
```

Comment:
Instead if excluding can you add java 17 for 3.x? I dont think it is beneficial to exclude the manifests

Reply:
Are you suggesting we adding another identical workflow with JDK 17 that includes check for 3.x manifest?